src/containers/App.jsx

```diff
@@ -1,13 +1,14 @@
-import {connect} from "react-redux";
+import { connect } from "react-redux";
 import AppComponent from "../components/App";
-import {fetchAllocations, fetchUsage, fetchLabUsage, logout} from "../actions";
+import { fetchAllocations, fetchUsage, fetchLabUsage, logout } from "../actions";
 import Cookies from "universal-cookie";
 
 const cookies = new Cookies();
 
 const mapStateToProps = (state) => {
-	return{
+	return {
 		Authorization: cookies.get("Authorization"),
+		// Authorization: state.user.Authorization,
```

Comment:
hmm we are directly reading from cookies now?

Reply:
oops, I think we can simply change the initial state declaration to set it from cookies. I'll test that and let you know.